python/tk_multi_loader/search_widget.py

```diff
@@ -70,7 +70,7 @@ def __setup_ui(self):
         icon.addPixmap(QtGui.QPixmap(":/res/clear_search.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
         clear_search.setIcon(icon)
         clear_search.setAutoRaise(True)
-        clear_search.clicked.connect(lambda editor=self.search: editor.setText(""))
+        clear_search.clicked.connect(lambda _: self.search.setText(""))
```

Comment:
The `clicked` callback first parameter is a boolean, so the code was broken. We're now ignoring the argument and simply clearing the text.